sct.py

```diff
@@ -1785,6 +1787,45 @@ def fetch_junit(runner_ip, backend):
     junit_xml_file.write_text(output.stdout.strip())
 
 
+@cli.command("upload", help="Upload arbitrary log/screenshot to s3 corresponding to the test_id")
+@click.option("--test-id", type=str, required=True)
+@click.option("--use-argus", type=bool, default=True)
```

Comment:
what's the use case of this flag ? cases you don't want to update argus about it ? for testing ?

Reply:
Could be useful in case you don't want to update argus for some reason / run failed to create in argus, but you still want to upload a file relevant to it.

Reply:
this one is a bit problematic, how do you can the command to disable  it ?

`hydra upload --no-use-argus` ? (I don't remember it by heart)

Reply:
yes, maybe better use `@click.option('--skip-argus', is_flag=True, default=False, help='Skip updating Argus')`

Reply:
Switched to using `--no-use-argus`, updated the `faq.md` with that as well - now you can disable argus upload with `--no-use-argus`